plugins/hypervisors/xenserver/src/com/cloud/hypervisor/xenserver/discoverer/XcpServerDiscoverer.java

```diff
@@ -410,6 +409,8 @@ else if (prodBrand.equals("XenServer") && prodVersion.equals("6.1.0"))
             return new XenServer610Resource();
         else if (prodBrand.equals("XenServer") && prodVersion.equals("6.5.0"))
             return new XenServer650Resource();
+        else if (prodBrand.equals("XenServer") && prodVersion.equals("7.0.0"))
+            return new XenServer700Resource();
```

Comment:
Looks good @karuturi . Thanks for the PR. Can you open it against my branch (syed/cloudstack:xenserver7) so that I can merge it in this PR?